c/httpserver.c

```diff
@@ -4518,6 +4533,36 @@ int runServiceThread(Socket *socket){
   return 0;
 }
 
+int runMonitoringThread(){
+#ifndef METTLE
+  int threadID; /* pthread_t threadID;  */
+  
+#ifdef DEBUG
+  printf("runMonitoringThread\n");
+#endif
+  fflush(stdout);
+  OSThread osThreadData;
+  OSThread *osThread = &osThreadData;
+  int createStatus = threadCreate(osThread,(void * (*)(void *))heartbeatMonitoringLoop, NULL);
```

Comment:
Way 1: It seem to work fine - create new thread - in thread while(true) {sleep(30)})